locales-pending/premium.ftl

```diff
@@ -247,3 +247,55 @@ false-door-test-content-part-two = Automatically remove data from sites trying t
 false-door-test-content-part-two-dashboard = Auto-delete data from sites trying to sell it.
 false-door-test-cta = Count me in
 false-door-test-popup-close = Close
+
+# High Risk Data Breaches
+
+high-risk-breach-heading = Here’s what to do
+high-risk-breach-subheading = This requires access to your sensitive info, so you’ll need to manually fix it.
+# Variables
+# $num_breaches is the number of breaches where the high risk data was found.
+high-risk-breach-summary = { $num_breaches ->
+  [one] It appeared in { $num_breaches } data breach:
+  *[other] It appeared in { $num_breaches } data breaches:
+}
+# Variables
+# $breach_name is the name of the breach where the high risk data was found.
+# $breach_date is the date when the breach occurred. 
+# An example of this string is Twitter on 13/09/18.
```

Comment:
The comment is misleading, since the plan is to localize the date. Could we fix the TODO in this PR, just to make sure it won't fall through the cracks? I assume we're already localizing dates in other parts of the code, or at least extracting the current locale.